README.md

```diff
@@ -1,104 +1,119 @@
-# Ethereum Contract
+# eth-abi
 
 [![Build Status](https://circleci.com/gh/ethereum/eth-abi.svg?style=shield)](https://circleci.com/gh/ethereum/eth-abi)
 [![Documentation Status](https://readthedocs.org/projects/eth-abi/badge/?version=latest)](https://readthedocs.org/projects/eth-abi/?badge=latest)
 [![PyPi version](https://img.shields.io/pypi/v/eth-abi.svg)](https://pypi.python.org/pypi/eth-abi)
 
+Python utilities for working with the Ethereum ABI.
 
-Python utilities for working with the Ethereum ABI
+## Overview
 
+The `eth-abi` library seeks to provide a set of low-level, extensible tools for
+interacting with the Ethereum ABI.  The most commonly used features of this
+library are briefly described below.  For complete documentation, visit
+[eth-abi.readthedocs.io](https://eth-abi.readthedocs.io/en/latest/).
 
 ## Installation
 
 ```sh
 pip install eth-abi
 ```
 
+## Decoding
 
+Decoding functions convert ABI-encoded binary data generated by the EVM into
+python values.
 
-## Documentation
+### eth\_abi.decode\_single(typ, data)
```

Comment:
I would wrap these titles in backticks to make the format as code blocks.  I think they are easier to read that way.

Reply:
The lack of backticks was on purpose actually.  The backticks didn't look great IMHO :(.  Also, the formatting was a bit nicer too when converted to rst.